cmd/grr/workflow.go

```diff
@@ -193,6 +193,26 @@ func previewCmd() *cli.Command {
 	return initialiseCmd(cmd, &opts)
 }
 
+func serveCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "serve <resources>",
+		Short: "Run Grizzly server",
+		Args:  cli.ArgsExact(1),
+	}
```

Comment:
This command needs a description and also, I'm not sure that `serve` is the correct command name for that. `serve-editor-proxy`, maybe?

Reply:
It is likely to do more than just the proxy. E.g. things like `grr show` could appear in the UI, reviewing YAML/JSON, converting between, managing contexts, and more could happen via the UI as much as cli.

Reply:
We'd like to merge then can discuss naming after, if needed.